src/FragmentCache.php

```diff
@@ -4,48 +4,48 @@
 
 namespace Yiisoft\Yii\Widgets;
 
-use function array_merge;
-use function ob_get_clean;
-use function ob_implicit_flush;
-use function ob_start;
+use RuntimeException;
 use Yiisoft\Cache\CacheInterface;
 use Yiisoft\Cache\Dependency\Dependency;
-
-use Yiisoft\View\DynamicContentAwareInterface;
-use Yiisoft\View\DynamicContentAwareTrait;
-use Yiisoft\View\WebView;
+use Yiisoft\View\Cache\CacheContent;
+use Yiisoft\View\Cache\DynamicContent;
 use Yiisoft\Widget\Widget;
 
-final class FragmentCache extends Widget implements DynamicContentAwareInterface
-{
-    use DynamicContentAwareTrait;
+use function ob_end_clean;
+use function ob_get_clean;
+use function ob_implicit_flush;
+use function ob_start;
 
-    private string $id;
+final class FragmentCache extends Widget
```

Comment:
It would be great to have phpdoc for it with a usage example.

Reply:
Done.